ui/src/components/TrafficPage.tsx

```diff
@@ -108,9 +111,31 @@ export const TrafficPage: React.FC<TrafficPageProps> = ({setAnalyzeStatus, onTLS
         }
     }
 
+    const openWebSocketCli = () => {
+        wsCli.current = new WebSocket("ws://localhost:8898/wsCli");
```

Comment:
what will happen here when user running "mizu view"? (it doesn't create the tappers nor create the socket)

Reply:
@IgorGov it just prints an error into console and it does not crash the React app.